Opentype/Fileformat/Cmap.hs

```diff
@@ -135,7 +134,7 @@ readCmapTable :: Strict.ByteString -> Either String CmapTable
 readCmapTable bs = do
   version <- index16 bs 0
   when (version /= 0) $
-    fail "unsupported cmap version."
+    Left "unsupported cmap version."
```

Comment:
`fail` is no longer a method of `Monad`, and there is no `MonadFail` instance for `Either String`.